app_server/relationships_functions.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from app_server.users_db_functions import *
+
+logger = logging.getLogger('gunicorn.error')
+
+def insert_new_friendship_request(user_email, new_friends_email, collection):
+	# Chequeo cero: que no sea yo mismo pidiendome amistad a mí mismo. Meme de spiderman
```

Comment:
![meme-spiderman](https://user-images.githubusercontent.com/12488006/86524454-46f04000-be51-11ea-82c4-83e2a015b9cc.jpg)

Reply:
THANK YOU